frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts

```diff
@@ -167,6 +181,16 @@ export class ResolutionInfo {
     return Math.log2(this.getLowestResolutionPowerOf2());
   }
 
+  getHighestResolution(): Vector3 {
+    // @ts-ignore
+    return this.getResolutionByPowerOf2(this.getHighestResolutionPowerOf2());
+  }
```

Comment:
I just noticed that this method isn't used anywhere. Only `getLowestResolution` is being used. 

But I think it is fine to keep this method in case someone wants the highest resolution in the future.